Google.Api.Gax.Rest/ClientBuilderBase.cs

```diff
@@ -99,6 +99,15 @@ public abstract class ClientBuilderBase<TClient>
         /// </remarks>
         public IHttpClientFactory HttpClientFactory { get; set; }
 
+        /// <summary>
+        /// Returns whether or not self-signed JWTs will be used over OAuth tokens when OAuth scopes are explicitly set.        
```

Comment:
Do we have a note about this only applying to SAs in GAPICs? If no, then that's fine.

Reply:
This comment is directly copy/pasted from the gRPC one :)